en/micro-integrator/docs/setup/deployment/k8s-pipeline/adding-test-cases.md

```diff
@@ -0,0 +1,107 @@
+# Adding test cases
+
+## Add your own test case to the Kubernetes Pipeline
+
+This document assumes that you have completed the instructions in the
+following pages.
+
+1.  [Quick Start Guide](/setup/deployment/k8s-pipeline/quick-start-guide/)
```

Comment:
Shall we call this "Kubernetes Pipeline Quick Start Guide" to differentiate it from other QSGs in this doc space?

Reply:
Since the user is inside the Pipeline documentation reading through it, it's intuitive that the QSG belongs to Pipeline documents